sdk-extensions/autoconfigure/src/main/java/io/opentelemetry/sdk/autoconfigure/MetricExporterConfiguration.java

```diff
@@ -132,32 +130,34 @@ static MetricExporter configureOtlpMetrics(
       throw new ConfigurationException("Unsupported OTLP metrics protocol: " + protocol);
     }
 
-    configureIntervalMetricReader(config, meterProvider, exporter);
+    configureIntervalMetricReader(config, sdkMeterProviderBuilder, exporter);
 
     return exporter;
   }
 
   private static void configureIntervalMetricReader(
-      ConfigProperties config, SdkMeterProvider meterProvider, MetricExporter exporter) {
-    IntervalMetricReaderBuilder readerBuilder =
-        IntervalMetricReader.builder()
-            .setMetricProducers(Collections.singleton(meterProvider))
-            .setMetricExporter(exporter);
+      ConfigProperties config,
+      SdkMeterProviderBuilder sdkMeterProviderBuilder,
+      MetricExporter exporter) {
+
     Duration exportInterval = config.getDuration("otel.imr.export.interval");
-    if (exportInterval != null) {
-      readerBuilder.setExportIntervalMillis(exportInterval.toMillis());
+    if (exportInterval == null) {
+      exportInterval = Duration.ofMinutes(1);
     }
-    IntervalMetricReader reader = readerBuilder.build().startAndRegisterGlobal();
-    Runtime.getRuntime().addShutdownHook(new Thread(reader::shutdown));
+    // Register the reader (which starts it), and use a global shutdown hook rather than
+    // interval reader specific.
```

Comment:
Registering the reader with the builder doesn't start it. It doesn't get started until the SdkMeterProvider is actually built, correct?

Reply:
Yeah, I'll update the comments.